streaming/src/util/streaming_util.h

```diff
@@ -4,94 +4,80 @@
 #include <string>
 #include <unordered_map>
 
+#include "ray/common/id.h"
 #include "util/streaming_logging.h"
 
 namespace ray {
 namespace streaming {
 
-enum class ConfigEnum : uint32_t {
-  QUEUE_ID_VECTOR = 0,
-  RECONSTRUCT_RETRY_TIMES,
-  RECONSTRUCT_TIMEOUT_PER_MB,
-  CURRENT_DRIVER_ID,
-  /// For direct call
-  CORE_WORKER,
-  SYNC_FUNCTION,
-  ASYNC_FUNCTION,
-  TRANSFER_MIN = QUEUE_ID_VECTOR,
-  TRANSFER_MAX = ASYNC_FUNCTION
-};
-}  // namespace streaming
-}  // namespace ray
-
-namespace std {
-template <>
-struct hash<::ray::streaming::ConfigEnum> {
-  size_t operator()(const ::ray::streaming::ConfigEnum &config_enum_key) const {
-    return static_cast<uint32_t>(config_enum_key);
-  }
-};
-
-template <>
-struct hash<const ::ray::streaming::ConfigEnum> {
-  size_t operator()(const ::ray::streaming::ConfigEnum &config_enum_key) const {
-    return static_cast<uint32_t>(config_enum_key);
-  }
-};
-}  // namespace std
-
-namespace ray {
-namespace streaming {
-
-class Config {
+class Util {
  public:
-  template <typename ValueType>
-  inline void Set(ConfigEnum key, const ValueType &any) {
-    config_map_.emplace(key, any);
-  }
+  static std::string Byte2hex(const uint8_t *data, uint32_t data_size);
 
-  template <typename ValueType>
-  inline void Set(ConfigEnum key, ValueType &&any) {
-    config_map_.emplace(key, any);
-  }
+  static std::string Hexqid2str(const std::string &q_id_hex);
 
-  template <typename ValueType>
-  inline boost::any &GetOrDefault(ConfigEnum key, ValueType &&any) {
-    auto item = config_map_.find(key);
-    if (item != config_map_.end()) {
-      return item->second;
+  template <typename T>
+  static std::string join(const T &v, const std::string &delimiter,
```

Comment:
Better to put those code into `.cc` if we don;t need inline for those methods call